docs/source/conf.py

```diff
@@ -43,11 +42,13 @@
 # built documents.
 #
 # The release is read from __init__ file and version is shortened release string.
-with open(os.path.join(os.path.dirname(__file__), "../../setup.py")) as setup_file:
-    for line in setup_file:
-        if "version=" in line:
-            release = line.split("=")[1].strip('", \n').strip("'")
-            version = release.rsplit(".", 1)[0]
+with open(os.path.join(os.path.dirname(__file__), "../../eolearn/__init__.py")) as init_file:
+    for line in init_file:
+        if line.find("__version__") >= 0:
+            release = line.split("=")[1].strip()
+            release = release.strip('"').strip("'")
+
+version = release.rsplit(".", 1)[0]
```

Comment:
Just noticed this, why is the version string here shortened?

Reply:
no clue, i asked myself this as well. Couldn't really figure it out so I didnt touch it since it's not really relevant in this scope